test/gpu/native/studies/chop/patches/GPU_aux.patch

```diff
@@ -0,0 +1,27 @@
+diff --git a/other_codes/chplGPU/modules/GPU_aux.chpl b/other_codes/chplGPU/modules/GPU_aux.chpl
+new file mode 100644
+index 0000000..98ad5f4
+--- /dev/null
++++ b/other_codes/chplGPU/modules/GPU_aux.chpl
+@@ -0,0 +1,9 @@
++module GPU_aux{
++
++	use CTypes;
++
++    require "headers/GPU_aux.h";
++
++    extern proc GPU_device_count(): c_int;
++
++}
```

Comment:
Are we adding this file in a patch?

Was there any major obstacle in dumping the contents here into `queens_GPU_single_locale.chpl`?

Reply:
We are adding. No major obstacle, I'm just copying (what appears to me to be) the structure of "`other_modules` contains a copy of `modules` with modifications", including `GPU_Aux.chpl`)